plugin/rpc/rpc.go

```diff
@@ -3,19 +3,30 @@ package rpc
 
 import (
 	"bytes"
+	"errors"
+	"fmt"
 	"io/ioutil"
 	"net/http"
+	"strings"
 )
 
 // "ExecuteRequest" takes in the raw json string and forwards it to the port
 func ExecuteRequest(jsonStr []byte, host string, port string) (string, error) {
+	if !strings.Contains(host, "http") {
```

Comment:
What if it's https?

Reply:
The contains function just looks for a substring, thus if the host contains https it will still pass. This is a QOL thing that fixes if someone does `localhost`